app/src/geo/google.js

```diff
@@ -3,12 +3,11 @@ let config = require('config');
 let inside = require('point-in-polygon');
 let _ = require('lodash');
 let NodeGeocoder = require('node-geocoder');
+
 let geocoder = NodeGeocoder({
```

Comment:
This makes geocoder === getCoords

Could you make the geolocate && getAddress both use geocoder too, to not have duplicate declarations